Gemfile.lock

```diff
@@ -112,14 +112,14 @@ GEM
     method_source (0.8.2)
     mime-types (1.25.1)
     mini_portile (0.6.0)
-    mongoid (3.1.6)
+    mongoid (3.1.7)
       activemodel (~> 3.2)
       moped (~> 1.4)
       origin (~> 1.0)
       tzinfo (~> 0.3.29)
     mongoid-paperclip (0.0.9)
       paperclip (>= 2.3.6)
-    moped (1.5.2)
+    moped (1.5.3)
```

Comment:
I was thinking the `moped` and `mongoid` changes would also get reverted for the same reason.